src/messages/textDocument_documentSymbol.cc

```diff
@@ -204,6 +204,12 @@ struct Handler_TextDocumentDocumentSymbol
             if (!def || !def->spell || def->is_local())
               continue;
           }
+          if (!params.all && sym.kind == SymbolKind::Type) {
```

Comment:
Do you use hierarchical document symbols?

Reply:
Thanks for implementing a better fix!